src/pages/_app.js

```diff
@@ -23,6 +23,16 @@ export default function App({ Component, pageProps }) {
 
   //-------------------------------------
 
+// Utility function to get currency symbol
```

Comment:
This helper function could be moved to a separate utils folder or similar. Just a suggestion.